src/image-copy/azext_imagecopy/_help.py

```diff
@@ -8,7 +8,7 @@
 
 helps['image copy'] = """
     type: command
-    short-summary: Copy a managed image (or vm) to other regions
+    short-summary: Copy a managed image (or vm) to other regions. Keep in mind that it requires the source disk to be available.
```

Comment:
Remove `Keep in mind that` to keep it short? You could probably remove the long summary as well since it basically repeats the short summary now.

Reply:
Good idea